layers/meta-resin-raspberrypi/recipes-kernel/linux-firmware/linux-firmware_git.bbappend

```diff
@@ -0,0 +1,14 @@
+# This .txt file is required for the firmware to work. It used to be
+# shipped, but now it's ommitted.
+#
+# XXX 2017.02.27 We believe this to be a bug, and the code below is a workaround
```

Comment:
I am fine with the tag. the message could be improved. But it's definitely a NIT.